src/caselawclient/Client.py

```diff
@@ -219,6 +219,58 @@ def is_document_published(self, judgment_uri):
         xml = etree.fromstring(content)
         return xml.text == "true"
 
+    def contains_sensitive_material(self, judgment_uri, sensitive=False):
+        uri = f"/{judgment_uri.lstrip('/')}.xml"
+        xquery_path = os.path.join(
+            ROOT_DIR, "xquery", "sensitive.xqy"
+        )
+        sensitive_value = "true" if sensitive else "false"
+        return self.eval(
+            xquery_path,
+            vars=f'{{"uri":"{uri}","sensitive":"{sensitive_value}"}}',
+            accept_header="application/xml",
+        )
+
+    def is_document_sensitive(self, judgment_uri):
+        uri = f"/{judgment_uri.lstrip('/')}.xml"
+        xquery_path = os.path.join(
+            ROOT_DIR, "xquery", "is-sensitive.xqy"
+        )
+
+        response = self.eval(
+            xquery_path, vars=f'{{"uri":"{uri}"}}', accept_header="multipart/mixed"
+        )
+
+        content = decoder.MultipartDecoder.from_response(response).parts[0].text
+        xml = etree.fromstring(content)
+        return xml.text == "true"
+
+    def supplemental_document(self, judgment_uri, supplemental=False):
```

Comment:
These methods are very similar, it would be good practice to reduce the duplication by combining them. You could create an method that all three of them call, which accepts an xquery filename and the other appropriate params. Same for the `is_document_*` methods too.